types/src/account_state/mod.rs

```diff
@@ -1,13 +1,19 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use crate::account_config::{AccountResource, ACCOUNT_RESOURCE_PATH};
-use anyhow::{Error, Result};
-use serde::export::Formatter;
-use serde::{Deserialize, Serialize};
-use std::collections::btree_map::BTreeMap;
-use std::convert::TryFrom;
-use std::fmt;
+use crate::{
+    account_config::{
+        AccountResource, ACCOUNT_RECEIVED_EVENT_PATH, ACCOUNT_RESOURCE_PATH,
+        ACCOUNT_SENT_EVENT_PATH,
+    },
+    discovery_set::{
+        DiscoverySetResource, DISCOVERY_SET_CHANGE_EVENT_PATH, DISCOVERY_SET_RESOURCE_PATH,
+    },
+    event::EventHandle,
+};
+use anyhow::{bail, Error, Result};
+use serde::{export::Formatter, Deserialize, Serialize};
+use std::{collections::btree_map::BTreeMap, convert::TryFrom, fmt};
 
 #[derive(Default, Deserialize, Serialize)]
 pub struct AccountState(BTreeMap<Vec<u8>, Vec<u8>>);
```

Comment:
What's the blob on index 1?

Reply:
? 

There is only .0 and it's a BTreeMap.

Reply:
I need to go to sleep.